### ISSUE TITLE
Sign in not functioning in macOS Ventura

### ISSUE THREAD

**Issue by jclem** [microsoft/vscode#155651]
Issue Type: <b>Bug</b>

In macOS Ventura Public Beta 1 (22A5295i), I can only get a sign-in to GitHub (for Settings Sync, Copilot, etc) to function by using the "copy code" method of signing in. Even then, as soon as I reload VSCode or quit-and-reopen VSCode, I've been logged out again.

VS Code version: Code 1.69.2 (3b889b090b5ad5793f524b5d1d39fda662b96a2a, 2022-07-18T16:32:39.000Z)
OS version: Darwin arm64 22.0.0
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M1 (8 x 24)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|4, 4, 4|
|Memory (System)|16.00GB (0.36GB free)|
|Process Argv|. --crash-reporter-id ace6edf5-ccbd-4d41-9b07-59403d086d6c|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (14)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-neovim|asv|0.0.88
vscode-tailwindcss|bra|0.8.6
better-toml|bun|0.3.2
vscode-eslint|dba|2.2.6
prettier-vscode|esb|9.5.0
codespaces|Git|1.9.3
copilot-labs|Git|0.2.221
copilot-nightly|Git|1.32.6264
fish-ide|lun|0.4.0
rose-pine|mvl|2.3.0
prisma|Pri|4.1.0
fish-vscode|sky|0.2.1
whichkey|VSp|0.11.3
viml|Xad|2.1.2

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstes627:30244334
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263cf:30335440
vscoreces:30445986
pythondataviewer:30285071
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
vsc1dst:30438360
pythonvs932:30410667
wslgetstarted:30449410
vscscmwlcmt:30465135
cppdebug:30492333
vscaac:30438847
pylanb8912cf:30529770
vsclangdc:30486549
841h4636:30530629

```

</details>

**Comment by jclem** [comment]
Looks like this [has been reported](https://github.com/microsoft/vscode/issues/154420), actually. It's difficult to tell as the others are all closed. Feel free to close this if I created a true duplicate.

**Comment by TylerLeonhardt** [comment]
Yes duplicate of #154420. The fix of this will be out early Aug.
